### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Daniil-Pankieiev**

### CODE REVIEW

File: app/managers.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self.connection = sqlite3.connect("cinema")
```
Comment (petrykivd) [inline app/managers.py:7]: cinema.db?

Comment (petrykivd) [inline app/managers.py:7]: _connection

File: app/managers.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self.connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self.connection.commit()
```
Comment (petrykivd) [inline app/managers.py:15]: _connection

**Review by petrykivd — CHANGES_REQUESTED** [review]

File: app/managers.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actor_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
```
Comment (fsocie7y) [inline app/managers.py:20]: It's better to follow SQL style guidelines even while writing raw SQL in Python. Therefore I'd suggest to start FROM clause from new line.

File: app/managers.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema")
```
Comment (fsocie7y) [inline app/managers.py:7]: You forgot to add a file extension, for example cinema.db3

File: app/managers.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        self._connection.execute(
+            f"UPDATE {self.table_name} "
+            f"SET first_name = ?, last_name = ? WHERE id = ?",
+            (name_to_update, surname_to_update, id_to_update)
+        )
+        self._connection.commit()
+
+    def delete(self, id_to_delete: int) -> None:
+        self._connection.execute(
+            f"DELETE FROM {self.table_name} WHERE id = ?", (id_to_delete,)
```
Comment (fsocie7y) [inline app/managers.py:39]: same about from
and in all cases u should add new line before SQL keyword

**Review by fsocie7y — CHANGES_REQUESTED** [review]

**Review by fsocie7y — APPROVED** [review]
GJ 🥇

**Pull request closed without merging**